src/pkg/utils/sbom.go

```diff
@@ -14,8 +15,8 @@ import (
 )
 
 // CreateSBOMArtifact creates sbom artifacts in the form of a tar archive
-func CreateSBOMArtifact(SBOMArtifactPathMap map[string]string) error {
-	out, err := os.Create(config.BundleSBOMTar)
+func CreateSBOMArtifact(SBOMArtifactPathMap map[string]string, bundleName string) error {
+	out, err := os.Create(fmt.Sprintf("%s-%s", bundleName, config.BundleSBOMTar))
```

Comment:
Should we do any sanitation of the `bundleName` here? This goes a bit beyond just the sbom.tar name, I noticed I could put a space in the bundle name and then the files generated end up with empty spaces in them. Maybe there's some naming rules and validation that I missed though

Reply:
ah yeah that's a really good point. let's chat about that with the team tomorrow. could be good a conversation for sanitizing overall.

Reply:
gonna create an issue for sanitizing the bundle name.